### PR TITLE
WT-5267 Store stop time pair in history store

### PR DESCRIPTION
**Pull request by kommiharibabu**
Stop time pair is required for every record in the history
store to represent the end of visibility of each record in
the history store. Store it in the key of the every history
record.

### CODE REVIEW

File: src/btree/bt_read.c [hunk 1]
```diff
@@ -178,9 +180,11 @@ __instantiate_lookaside(WT_SESSION_IMPL *session, WT_REF *ref)
     WT_PAGE *page;
     WT_PAGE_LOOKASIDE *page_las;
     WT_UPDATE *mod_upd, *upd;
-    wt_timestamp_t durable_timestamp, durable_timestamp_tmp, las_timestamp, las_timestamp_tmp;
+    wt_timestamp_t durable_timestamp, durable_timestamp_tmp, las_stop_timestamp,
+      las_stop_timestamp_tmp, las_timestamp, las_timestamp_tmp;
```
Comment (tetsuo-cpp) [inline src/btree/bt_read.c]: Nit: The convention is to not wrap declarations like that.
```
wt_timestamp_t durable_timestamp, durable_timestamp_tmp, las_stop_timestamp;
wt_timestamp_t las_stop_timestamp_tmp, las_timestamp, las_timestamp_tmp;
```

**Comment by bvpvamsikrishna** [comment]
@quchenhao, please review this pull request and try to understand the changes.

File: src/cache/cache_las.c [hunk 2]
```diff
@@ -797,6 +813,13 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
             WT_ERR(cursor->update(cursor));
             ++insert_cnt;
 
+            /*
+             * Store the current update comit timestamp and transaction id, these are the stop
```
Comment (tetsuo-cpp) [inline src/cache/cache_las.c]: Typo: `comit => commit`.

Comment (kommiharibabu) [reply src/cache/cache_las.c]: Fixed.

**Pull request merged**